util/unzip_nix_test.go

```diff
@@ -45,21 +45,23 @@ func TestUnzipOtherUser(t *testing.T) {
 // TestUnzipFileModTime checks to make sure after unpacking zip file the file
 // modification time is "now" and not the original file time.
 func TestUnzipFileModTime(t *testing.T) {
-	now := time.Now().UnixNano()
-	t.Logf("Now: %d", now)
+	now := time.Now()
+	t.Logf("Now: %s", now)
 	destinationPath := TempPath("", "TestUnzipFileModTime.")
 	err := Unzip(testZipPath, destinationPath, testLog)
 	require.NoError(t, err)
 
 	fileInfo, err := os.Stat(filepath.Join(destinationPath, "test"))
 	require.NoError(t, err)
-	diffDir := now - fileInfo.ModTime().UnixNano()
-	t.Logf("Diff (dir): %d", diffDir)
-	assert.True(t, diffDir >= 0)
+	dirMod := fileInfo.ModTime()
+	diffDir := dirMod.Sub(now)
```

Comment:
old test was checking that `now >= fileInfo.ModTime()`, but we actually want `fileInfo.ModTime() >= now`, right? (i.e., the unpacked file was created after we looked at the clock).

Reply:
I see, on Linux the file/dir mod times seem to be *before* `now`, which is weird

Reply:
Maybe it's a precision thing, like the Linux timestamps are rounded down to the nearest millisecond?